ibis/backends/dask/dispatch.py

```diff
@@ -0,0 +1,21 @@
+from multipledispatch import Dispatcher
+
+import ibis.backends.pandas.core as core_dispatch
+import ibis.backends.pandas.dispatch as pandas_dispatch
+from ibis.backends.dask.trace import TraceTwoLevelDispatcher
+
+execute_node = TraceTwoLevelDispatcher('dask_execute_node')
```

Comment:
Can you please change the names passed to the dispatchers? This way it will look like all the others. 

And then update `test_core.py`?